webrtc-stats.html

```diff
@@ -1690,56 +1694,13 @@ <h2>
         <h3>
           <dfn>RTCMediaStreamTrackStats</dfn> dictionary
         </h3>
-        <p>
-          An RTCMediaStreamTrackStats object represents the stats about one attachment of a
-          MediaStreamTrack to the PeerConnection object for which one calls getStats.
-        </p>
-        <p>
-          It appears in the stats as soon as it is attached (via addTrack, via addTransceiver, via
-          ReplaceTrack on an RTPSender object, or via being created on an RTPReceiver object).
-        </p>
-        <p>
-          If an outgoing track is attached twice (via addTransceiver or ReplaceTrack), there will
-          be two RTCMediaStreamTrackStats objects, one for each attachment. They will have the same
-          "trackIdentifier" attribute, but different "id" attributes.
-        </p>
-        <p>
-          If the track is detached from the PeerConnection (via removeTrack or via replaceTrack),
-          it continues to appear, but with the "detached" member set to True.
-        </p>
```

Comment:
It would be helpful to have a paragraph here saying this is the dictionary that RTC[Audio/Video]StreamTrackStats inherits from, that there are no instances of this dictionary directly.

Reply:
Isn't this clear from the WebIDL? There's precedent here with e.g. RTCReceivedRTPStreamStats, so I recommend a separate issue.